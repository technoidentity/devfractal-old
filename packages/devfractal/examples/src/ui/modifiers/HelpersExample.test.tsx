import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Helpers } from './HelpersExample'

test('Helpers Example', () => {
  const component: ReactTestRenderer = renderer.create(<Helpers />)
  expect(component.toJSON()).toMatchSnapshot()
})
