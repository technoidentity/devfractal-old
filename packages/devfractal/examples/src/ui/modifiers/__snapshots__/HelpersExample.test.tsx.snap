// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Helpers Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Spacing
    </h1>
    <div
      className="box"
    >
      <div
        className="has-text-weight-bold"
      >
        is-paddingLess
      </div>
      <div
        className="control"
      >
        <button
          className="button is-paddingless"
          type="button"
        >
          Button
        </button>
      </div>
      <div
        className="has-text-weight-bold"
      >
        with padding
      </div>
      <div
        className="control"
      >
        <button
          className="button"
          type="button"
        >
          Button1
        </button>
      </div>
      <div
        className="has-text-weight-bold"
      >
        is-marginLess
      </div>
      <div
        className="control"
      >
        <button
          className="button is-marginless"
          type="button"
        >
          Button
        </button>
      </div>
      <div
        className="has-text-weight-bold"
      >
        with margin
      </div>
      <div
        className="control"
      >
        <button
          className="button"
          type="button"
        >
          Button
        </button>
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Float
    </h1>
    <div
      className="box"
    >
      <div
        className="has-text-weight-bold"
      >
        is-pulled-left
      </div>
      <button
        className="button"
        type="button"
      >
        Button1
      </button>
      <button
        className="button is-pulled-left"
        type="button"
      >
        Pulled Left Button 2
      </button>
      <button
        className="button"
        type="button"
      >
        Button3
      </button>
      <div
        className="has-text-weight-bold"
      >
        is-pulled-right
      </div>
      <button
        className="button"
        type="button"
      >
        Button1
      </button>
      <button
        className="button is-pulled-right"
        type="button"
      >
        Pulled Right Button 2
      </button>
      <button
        className="button"
        type="button"
      >
        Button3
      </button>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Other
    </h1>
    <div
      className="box"
    >
      <div
        className="has-text-weight-bold"
      >
        is-shadowLess
      </div>
      <div
        className="control"
      >
        <button
          className="button is-shadowless"
          type="button"
        >
          Button
        </button>
      </div>
      <div
        className="has-text-weight-bold"
      >
        with shadow
      </div>
      <div
        className="control"
      >
        <button
          className="button"
          type="button"
        >
          Button
        </button>
      </div>
      <div
        className="has-text-weight-bold"
      >
        is-unselectable
      </div>
      <div
        className="is-unselectable"
      >
        unselectable text
      </div>
      <div
        className="has-text-weight-bold"
      >
        selectable
      </div>
      <div
        className=""
      >
        selectable text
      </div>
      <div
        className="has-text-weight-bold"
      >
        is-invisible
      </div>
      <div
        className="control"
      >
        <button
          className="button is-invisible"
          type="button"
        >
          Button
        </button>
      </div>
      <div
        className="has-text-weight-bold"
      >
        visible
      </div>
      <div
        className="control"
      >
        <button
          className="button"
          type="button"
        >
          Button
        </button>
      </div>
      <div
        className="has-text-weight-bold"
      >
        is-radiusless
      </div>
      <div
        className="control"
      >
        <button
          className="button is-radiusless"
          type="button"
        >
          Button
        </button>
      </div>
      <div
        className="has-text-weight-bold"
      >
        with radius
      </div>
      <div
        className="control"
      >
        <button
          className="button"
          type="button"
        >
          Button
        </button>
      </div>
    </div>
  </section>,
]
`;
